app/institutions/dashboard/-components/chart-kpi-wrapper/chart-kpi/template.hbs

```diff
@@ -0,0 +1,60 @@
+<div
+    local-class='chart-container {{if (is-mobile) 'mobile'}}'
+    ...attributes 
+>
+    <div local-class='top-container'>
+        <div
+            data-test-chart
+            local-class='ember-chart'
+        >
+            <EmberChart
+                @type={{@data.chartType}}
+                @options={{this.chartOptions}}
+                @data={{this.chartData}}
+            />
+        </div>
+    </div>
+    <div local-class='bottom-container'>
+        <div local-class='title-container'>
+            <div data-test-chart-title local-class='title'>{{@data.title}}</div>
+            <div local-class='button-container'>
+                <Button
+                    data-analytics-name='Expand additional data'
+                    data-test-expand-additional-data
+                    @layout='fake-link'
+                    aria-expanded={{this.expanded}}
```

Comment:
+1 for using `aria-expanded`! I recall `aria-expanded` needs a `aria-controls` to indicate what element this button controls, so you'll need to add a local variable using the `{{#let (unique-id) as |expandableElementId|}}`, add an `id={{expandableElementId}}` to the `<div local-class='expanded-data-container'>` (ln 33), and add `aria-controls={{expandableElementId}}` to this Button

Reply:
Looks like this button needs an aria-label as well, since there is no text for screen readers to use